test/configCases/target/amd-unnamed/index.js

```diff
@@ -1,7 +1,7 @@
 it("should run", function() {});
 
 it("should name define", function() {
-	var fs = require("fs");
+	var fs = require("node:fs");
```

Comment:
Please add new tests, don't modify exists

Reply:
As I wrote above, I don't understand how to add new tests here - I tried :)